internal/test/clusterinfo/openshift.go

```diff
@@ -0,0 +1,66 @@
+package clusterinfo
+
+import (
+	"context"
+	"fmt"
+
+	v1 "github.com/openshift/api/config/v1"
+	clientset "github.com/openshift/client-go/config/clientset/versioned"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client/config"
+)
+
+// OpenShift is an Client struct which will be used for all OpenShift related functions to interact with the existing
+// Cluster.
+type OpenShift struct {
+	Client *clientset.Clientset
+}
+
+// GetOpenShift creates client for the current OpenShift cluster. If KUBECONFIG env var is set, it is used to
+// create client, otherwise it uses in-cluster config.
+func GetOpenShift() (*OpenShift, error) {
```

Comment:
Shouldnt this be named something like `NewOpenShift`? Since it is constructing an instance of the `OpenShift` struct.